### DISCUSSION TITLE
Think about our clean code rules and its related documentation

### DISCUSSION THREAD

**Discussion by vlo-rte** [opfab/operatorfabric-core#3701] [Ideas]
Think about our existing rules and new rules which could be useful for us.
Order these rules in a documentation.
Make the documentation more easily accessible.

People can propose rule(s) and we can discuss it (them) of course.

**Comment by freddidierRTE** [comment]
Interesting rules to have a look : https://github.com/ryanmcdermott/clean-code-javascript
